pkg/cloud/libvirt/actuators/machine/utils/domain.go

```diff
@@ -497,9 +497,6 @@ func CreateDomain(name, ignKey, volumeName, hostName, networkInterfaceName, netw
 
 	glog.Infof("setDisks")
 	VolumeKey := baseVolumePath + volumeName
-	if volumeName == "" {
```

Comment:
we should ideally narrow down the number of parameters injected, encapsulate them into a struct, validate the inputs/error when relevant and unit test it. I understand might be out of the scope for your current work

Reply:
that would be perfect, but it is a bit out of scope